main.go

```diff
@@ -62,6 +62,7 @@ const (
 	dbNameEnv     = "DB_NAME"
 	dbUserEnv     = "DB_USER"
 	dbPasswordEnv = "DB_PASSWORD"
+	dbDSNEnv      = "DSN"
```

Comment:
Should we change this to `DB_DSN`? @niravparikh05 @akshay196

Reply:
We can move this to top to keep it consistent.
/nit

Reply:
yeah I guess we could change it to DB_DSN to keep it consistent with other database attributes. However same has to be modified in helm-charts as well